torchinductor/scheduler.py

```diff
@@ -1004,7 +1004,10 @@ def remove_kernel_local_buffers(self):
         # If all the uses of this buffer are also in self.pending_buffer_names,
         # it means the buffer becomes kernel-local after fusion
         for name in self.pending_buffer_names:
-            if name in V.kernel.must_keep_buffers:
+            if (
```

Comment:
I wonder if it's semantically cleaner to just add `input_buffers` into `must_keep_buffers`?